gordo/client/__init__.py

```diff
@@ -1 +1,6 @@
-__version__ = "0.1.0"
+from pkgutil import extend_path
+
+from gordo.client.client import Client
+from gordo.client.utils import influx_client_from_uri
+
+__path__ = extend_path(__path__, __name__)
```

Comment:
Is it really required? I was sure [PEP-420](https://www.python.org/dev/peps/pep-0420/)-compliant package structure is enough.

Reply:
yes, without it i cant import gordo.client from gordo

Reply:
looks weird. at least it deserves a comment with an explanation of why we did this

Reply:
done